OpenXmlPowerTools.Tests/Word/DocumentAssemblerTests.cs

```diff
@@ -206,6 +207,42 @@ public void DA102_Throws(string name, string data)
                 });
         }
 
+        [Theory]
+        [InlineData("DA-TemplateMaior.docx", "DA-TemplateMaior.xml", false)]
+        public void DATemplateMaior(string name, string data, bool err)
+        {
+            DA101(name, data, err);
+            var assembledDocx = new FileInfo(Path.Combine(TempDir, name.Replace(".docx", "-processed-by-DocumentAssembler.docx")));
+            var afterAssembling = new WmlDocument(assembledDocx.FullName);
+
+            var descendants = afterAssembling.MainDocumentPart.Value;
+
+            if (descendants.Contains(">"))
+            {
+                Assert.Fail("Found > on text");
```

Comment:
There is not such thing as `Asset.Fail`, can you please make it compilable?

Reply:
I updated to xunit 2.4.2, now I changed back to 2.4.1